index.html

```diff
@@ -2139,8 +2139,8 @@ <h3>Retrieve Verification Method</h3>
 <a href="#INVALID_VERIFICATION_METHOD">INVALID_VERIFICATION_METHOD</a>.
           </li>
           <li>
-If <var>verificationMethod</var> is not associated with the
-<var>verificationRelationship</var> array in the <var>controllerDocument</var>,
+If the [=verification method=] is not associated with the
```

Comment:
There can be multiple arrays in the controller document containing verification relationships, correct?  If so, please change "the" to "a".

Reply:
```suggestion
If the [=verification method=] is not associated with a
```